migrations/m3_acp_data.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ *
+ * Ideas extension for the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ */
+
+namespace phpbb\ideas\migrations;
+
+class m3_acp_data extends \phpbb\db\migration\migration
+{
+	public function effectively_installed()
+	{
+		return isset($this->config['ideas_forum_id']) || isset($this->config['ideas_poster_id']);
```

Comment:
Lets be a little cooler here and instead do:
`return $this->config->offsetExists('ideas_forum_id') || $this->config->offsetExists('ideas_poster_id');`